### DISCUSSION TITLE
Running 0.15 and can't get media to play in HA (was happening in 0.14 too)

### DISCUSSION THREAD

**Discussion by flyize** [blakeblackshear/frigate#15276] [Ask A Question] [answered]
I have the Frigate integration updated to 5.5.1 and am running 0.15-beta2. Also running latest version of HA. This is what shows when I try to view a clip. There don't seem to be any errors in the  Frigate logs. What could I be missing?

![image](https://github.com/user-attachments/assets/baa95359-0b34-46f4-8d60-299754b289a5)

**Comment by NickM-27** [comment, marked as answer]
This is a known bug in HA, fixed in 2024.12

**Comment by flyize** [comment]
Great news. Thank you!
